File: nodejs/s3-client-mrap.js
const { S3Client, ListObjectsCommand } = require("@aws-sdk/client-s3");

async function start() {
    const client = new S3Client({ region: "us-east-1" });
    const bucketParams = { Bucket: "arn:aws:s3::018821064628:accesspoint/mnwtgptndzniq.mrap" };
    const command = new ListObjectsCommand(bucketParams);
    const response = await client.send(command);
    console.log(response.Contents);
}

start();

File: python/s3-client-mrap.py
import boto3
import boto3.session

# Create your own session
my_session = boto3.session.Session()

client = my_session.resource('s3', region_name='sa-east-1')

bucket = client.Bucket('arn:aws:s3::018821064628:accesspoint/mnwtgptndzniq.mrap')
for obj in bucket.objects.all():
    print(obj.key)